Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/.cache\n/dist\n/node_modules\n/package-lock.json\n/.DS_Store\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	
+++ b/.gitignore	
@@ -1,5 +1,6 @@
 /.cache
 /dist
+/.parcel-cache
 /node_modules
 /package-lock.json
 /.DS_Store
